src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.css

```diff
@@ -14,12 +31,77 @@
 	text-transform: uppercase;
 }
 
+::ng-deep .forminput .forminput-x-inputs input,
+::ng-deep .forminput select,
+::ng-deep .forminput textarea {
+    border: 0.5px solid var(--color-purple) !important;
+	border-radius: 0.25rem !important;
+}
+
+::ng-deep .forminput .forminput-x-sublabel {
+	height: 2.5rem
+}
+
+::ng-deep .dropzone {
+	background-color: white !important;
+	text-align: center;
+	border: 1px solid var(--color-purple) !important;
+	height: 20rem
+}
+
+
+@media (max-width: 480px) {
+	::ng-deep .dropzone .dropzone-x-info1,
+	::ng-deep .dropzone .dropzone-x-info2 {
+		display: block !important;
+	}
+}
+
 .section {
 	background-color: #f1f0ff;
 	border-radius: 25px;
-	padding: 2em;
+	margin-bottom: 2em;
+	padding: 1em;
 }
 
+
 li::marker {
 	color: var(--color-blue);
 }
+
+.ng-autocomplete {
+	width:100%;
+	max-width: 500px;
+  }
+
+.competencyDuration {
```

Comment:
This class has some duplicate attributes `font-family`, `line-height`...

Reply:
Deleted the duplicates